service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/auth/AuthenticationType.java

```diff
@@ -12,6 +14,8 @@
 public enum AuthenticationType
 {
     SAS,
```

Comment:
Looks like the SAS enum value is also incorrect based on the service's swagger doc
```suggestion
    @SerializedName("sas")
    SAS,
```

Reply:
I have tested it with both **with** `@SerializedName("sas")` and **without** `@SerializedName("sas")` cases, it works in both cases